src/screens/Schedule.js

```diff
@@ -14,6 +15,7 @@ class Schedule extends Component {
     return (
         <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
             <Text>Schedule!</Text>
+            <CustomButton type="submit" buttonColorBackground="#FF7A67">Login</CustomButton>
```

Comment:
Remove display test component (won't be needed in this page)

Reply:
I did not submit schedule.js, but I will remove it on my end.

Reply:
Can you first rebase your branch onto develop? If it is up to date, reviewers only see the changes made by you.